state-synchronizer/src/lib.rs

```diff
@@ -13,7 +13,7 @@ pub use synchronizer::{StateSyncClient, StateSynchronizer};
 
 mod chunk_request;
 mod chunk_response;
-mod coordinator;
+pub mod coordinator;
```

Comment:
so ugly

Reply:
I can do little unless someone refactor this part to be a trait.